auth-password-store.el

```diff
@@ -52,6 +52,10 @@ See `auth-source-search' for details on SPEC."
 
 (defun auth-pass--build-result (host port user)
   "Build auth-pass entry matching HOST, PORT and USER."
+  (if (string-match-p (regexp-quote ":") host)
```

Comment:
Can you please tell me which case is not working for you the way the you want? How can I reproduce your problem?

Reply:
Please pay attention that `host` often contains the protocol too: `host = "https://gitlab.com:334"`. Your code will break in these cases.

Can you please tell me which case is not working for you the way the you want? How can I reproduce your problem?

Reply:
Greetings. 

In my **.password-store** I have an entry with the credentials of a webdav server for example: **webdav/dav.test.com**, inside this entry I have the following information: 
```
dav.test.com 
----------------- 
password 
URL: dav.test.com 
username: user@test.com 
port: https
```
and use org-caldav. When I run **org-caldav-sync**, emacs looks for a password for the host: **dav.test.com:443**, because the port is included the URL does not match the corresponding entry